assets/js/modules/analytics-4/utils/validation.js

```diff
@@ -58,5 +58,5 @@ export function isValidPropertySelection( value ) {
  * @return {boolean} TRUE if the web data stream ID is valid, otherwise FALSE.
  */
 export function isValidWebDataStreamID( webDataStreamID ) {
-	return typeof webDataStreamID === 'string' && /\d+/.test( webDataStreamID );
+	return typeof webDataStreamID === 'string' && /^[a-zA-Z0-9]+$/.test( webDataStreamID );
```

Comment:
See above, the `isValidWebDataStreamID` implementation from before is actually correct. What we need here instead is a new `isValidMeasurementID` function.

Reply:
Updated.